mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,21 @@
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
+        <intent-filter>
+            <data android:scheme="${SCHEME}" />
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+        </intent-filter>
+        <intent-filter android:autoVerify="true">
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+
+          <data android:scheme="http" />
+          <data android:scheme="https" />
+          <data android:host="${HOST}" />
+        </intent-filter>
```

Comment:
I think we also need to add navigation logic for some screens, for example sign-in, sign-up

Reply:
Example 
![image](https://user-images.githubusercontent.com/43109766/183289928-c0cb784b-3622-47cb-85fb-728ae7672591.png)

Reply:
Thanks i will try to do it